pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -58,6 +60,29 @@ def __enter__(self):
         print(msg)
         bokchoy_utils.check_services()
 
+        if not self.testsonly:
+            self.prepare_bokchoy_run()
+
+        msg = colorize('green', "Waiting for servers to start...")
+        print msg
+        bokchoy_utils.wait_for_test_servers()
+        if self.serversonly:
+            self.interruptable_prompt()
+
+    def __exit__(self, exc_type, exc_value, traceback):
```

Comment:
Do we want to run all of this when you specify --testsonly? I feel like it should do "no setup" and "no teardown"...